index.rst

```diff
@@ -50,6 +50,63 @@
 
    This document explains the procedure of what to do when a new filter is added (or going to be added) to the system. Several steps are required in order to ensure ingestion by the Butler such that the data can be served.
 
+
+Introduction
+============
+
+Adding a filter or disperser to LATISS is a multi-step procedure that spans the DM and T&S subsystems.
+This technote is a list of things to do in order to have the data be taken with the correct metadata, as well as what it takes to get the files ingested on the summit.
+
+.. important::
+
+   The most time critical step is updating filters.py (see below) and having it be built in a regular weekly that is accessible from the summit environment.
+   Weeklies are built on Thursday nights, and can't be deployed until Friday at the earliest.
```

Comment:
DM builds weeklies Wed night/Thu morning, but I'm not sure how long it takes Tiago to pick that up.  I thought it was pretty rapid.